third_party/gems/bundle-env.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# --- begin runfiles.bash initialization ---
+# Copy-pasted from Bazel's Bash runfiles library (tools/bash/runfiles/runfiles.bash).
+set -euo pipefail
```

Comment:
This affects the script globally. Even if it was copy / pasted, can we move it to the top of the file?

Reply:
Happy to move that part out of the region copied from bazel. I had left it inside to ensure that it was consistent with what the bazel docs were recommending (https://github.com/bazelbuild/bazel/blob/master/tools/bash/runfiles/runfiles.bash).

Reply:
Also, note that version is newer than what we get with bazel-0.21.0. We're using the version that they're referring to as `version 1` in that file.

Reply:
^ even better to link to a SHA then.

Reply:
I've modified all the snippets to refer to the SHA for 0.21.0.